src/gz.rs

```diff
@@ -862,6 +863,17 @@ impl Header {
     pub fn mtime(&self) -> u32 {
         self.mtime
     }
+
+    /// Returns the most recent modification time as a datetime if time stamp is available.
```

Comment:
the interpretation of `mtime` is already in the associated method,  so do I also duplicate the parts about year 2038 and local vs universal,  or just the meaning of it as time since the epoch?